src/manifest.json

```diff
@@ -7,6 +7,7 @@
   "permissions": [
     "notifications",
     "storage",
+    "activeTab",
```

Comment:
We will need to prepare explanation when push to web store, just to take in account cc: @beemi